test/modules/Relayer/Program.cs

```diff
@@ -100,23 +113,30 @@ static async Task<MessageResponse> ProcessAndSendMessageAsync(Message message, o
                     BatchId = batchId,
                     SequenceNumber = sequenceNumber
                 };
-                await ModuleUtil.ReportTestResultAsync(testResultReportingClient, Logger, testResultReceived);
-                Logger.LogInformation($"Successfully received message: trackingid={trackingId}, batchId={batchId}, sequenceNumber={sequenceNumber}");
 
-                byte[] messageBytes = message.GetBytes();
-                var messageCopy = new Message(messageBytes);
-                messageProperties.ForEach(kvp => messageCopy.Properties.Add(kvp));
-                await moduleClient.SendEventAsync(Settings.Current.OutputName, messageCopy);
+                await ExecuteWithRetry(
```

Comment:
why do we need retry for test result reporting?

Reply:
I actually don't think we do anymore. Artifact from testing. Removed.